pkg/engine/virtualmachinescalesets.go

```diff
@@ -718,3 +720,9 @@ func getVMSSDataDisks(profile *api.AgentPoolProfile) *[]compute.VirtualMachineSc
 	}
 	return &dataDisks
 }
+
+func addCustomTagsToVMScaleSets(cs *api.ContainerService, vm *compute.VirtualMachineScaleSet) {
```

Comment:
It is happening here, @CecileRobertMichon :)

Reply:
this is still for MasterProfile

Reply:
we're missing the agentPoolProfile tags

Reply:
https://github.com/Azure/aks-engine/blob/master/pkg/engine/virtualmachines.go#L361 for VMAS

Reply:
https://github.com/Azure/aks-engine/blob/master/pkg/engine/virtualmachinescalesets.go#L379 for VMSS

Reply:
Oh, I see! Sorry about that! I'll fix that!

Reply:
Also done!